File: o2ios/O2Platform/App/Work-工作/m/WorkInfoResData.swift
//
//  WorkInfoResData.swift
//  O2Platform
//
//  Created by FancyLou on 2020/9/4.
//  Copyright © 2020 zoneland. All rights reserved.
//

import Foundation
import HandyJSON

class WorkInfoRes: NSObject,DataModel {
    @objc var id: String?
    @objc var job: String?
    @objc var title: String?
    @objc var startTime: String?
    @objc var startTimeMonth: String?
    @objc var creatorPerson: String?
    @objc var creatorIdentity: String?
    @objc var creatorUnit: String?
    @objc var application: String?
    @objc var applicationName: String?
    @objc var applicationAlias: String?
    @objc var process: String?
    @objc var processName: String?
    @objc var processAlias: String?
    @objc var activity: String?
    @objc var activityType: String?
    @objc var activityName: String?
    @objc var activityAlias: String?
    @objc var activityDescription: String?
    @objc var activityToken: String?
    @objc var activityArrivedTime: String?
    @objc var serial: String?
    @objc var workCreateType: String?
    @objc var workStatus: String?
    @objc var manualTaskIdentityText: String?
    @objc var form: String?
    @objc var destinationRoute: String?
    @objc var destinationRouteName: String?
    @objc var destinationActivityType: String?
    @objc var destinationActivity: String?
    
    
    required override init() {
        
    }
}

class WorkInfoResData: NSObject,DataModel {
    @objc var work: WorkInfoRes?
    
    required override init() {
        
    }
}

File: o2android/o2_auth_sdk/src/main/java/net/zoneland/x/bpm/mobile/v1/zoneXBPM/model/bo/api/o2/WorkInfoResData.kt
package net.zoneland.x.bpm.mobile.v1.zoneXBPM.model.bo.api.o2


/**
 * Created by fancyLou on 2020-09-04.
 * Copyright © 2020 O2. All rights reserved.
 */

data class WorkInfoResData(
        var work: WorkInfoRes? ,
        var workLogList: List<WorkLog> = ArrayList()
)

data class WorkInfoRes(
    var id: String = "",
    var job: String = "",
    var title: String = "",
    var startTime: String = "",
    var startTimeMonth: String = "",
    var creatorPerson: String = "",
    var creatorIdentity: String = "",
    var creatorUnit: String = "",
    var application: String = "",
    var applicationName: String = "",
    var applicationAlias: String = "",
    var process: String = "",
    var processName: String = "",
    var processAlias: String = "",
    var activity: String = "",
    var activityType: String = "",
    var activityName: String = "",
    var activityAlias: String = "",
    var activityDescription: String = "",
    var activityToken: String = "",
    var activityArrivedTime: String = "",
    var serial: String = "",
    var workCreateType: String = "",
    var workStatus: String = "",
    var manualTaskIdentityText: String = "",
    var form: String = "",
    var destinationRoute: String = "",
    var destinationRouteName: String = "",
    var destinationActivityType: String = "",
    var destinationActivity: String = ""
)